quarkus/integration-tests/server/src/test/resources/configuration/log4j2.xml

```diff
@@ -0,0 +1,87 @@
+    <Properties>
+        <Property name="path">${sys:infinispan.server.log.path}</Property>
+        <Property name="accessLogPattern">%X{address} %X{user} [%d{dd/MMM/yyyy:HH:mm:ss Z}] &quot;%X{method} %m %X{protocol}&quot; %X{status} %X{requestSize} %X{responseSize} %X{duration}%n</Property>
+    </Properties>
+    <Appenders>
+        <!-- Colored output on the console -->
+        <Console name="STDOUT">
+            <PatternLayout pattern="%highlight{%d{HH:mm:ss,SSS} %-5p [%c{1.}] %m%throwable}{INFO=normal, DEBUG=normal, TRACE=normal}%n"/>
+        </Console>
+
+        <!-- Rolling file -->
+        <RollingFile name="FILE" createOnDemand="true"
+                     fileName="${path}/server.log"
+                     filePattern="${path}/server.log.%d{yyyy-MM-dd}-%i">
+            <Policies>
+                <OnStartupTriggeringPolicy />
+                <SizeBasedTriggeringPolicy size="100 MB" />
+                <TimeBasedTriggeringPolicy />
+            </Policies>
+            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{1.}] %m%throwable%n"/>
+        </RollingFile>
+
+        <!-- Rolling JSON file, disabled by default -->
+        <RollingFile name="JSON-FILE" createOnDemand="true"
+                     fileName="${path}/server.log.json"
+                     filePattern="${path}/server.log.json.%d{yyyy-MM-dd}-%i">
+            <Policies>
+                <OnStartupTriggeringPolicy />
+                <SizeBasedTriggeringPolicy size="100 MB" />
+                <TimeBasedTriggeringPolicy />
+            </Policies>
+            <JsonLayout compact="true" eventEol="true" stacktraceAsString="true">
+                <KeyValuePair key="time" value="$${date:yyyy-MM-dd'T'HH:mm:ss.SSSZ}" />
+            </JsonLayout>
+        </RollingFile>
+
+        <!-- Rolling HotRod access log, disabled by default -->
+        <RollingFile name="HR-ACCESS-FILE" createOnDemand="true"
+                     fileName="${path}/hotrod-access.log"
+                     filePattern="${path}/hotrod-access.log.%i">
+            <Policies>
+                <SizeBasedTriggeringPolicy size="100 MB" />
+            </Policies>
+            <PatternLayout pattern="${accessLogPattern}"/>
+        </RollingFile>
+        <!-- Rolling REST access log, disabled by default -->
+        <RollingFile name="REST-ACCESS-FILE" createOnDemand="true"
+                     fileName="${path}/rest-access.log"
+                     filePattern="${path}/rest-access.log.%i">
+            <Policies>
+                <SizeBasedTriggeringPolicy size="100 MB" />
+            </Policies>
+            <PatternLayout pattern="${accessLogPattern}"/>
+        </RollingFile>
+    </Appenders>
+
+    <Loggers>
+        <Root level="INFO">
+            <AppenderRef ref="STDOUT" level="TRACE"/>
+
+            <!-- Uncomment just one of the two lines bellow to use alternatively JSON logging or plain-text logging to file-->
+            <AppenderRef ref="FILE"/>
```

Comment:
Should we not swap these around so TRACE goes to the file instead and the console is INFO?

Reply:
Maybe... I just copied this file from `server/tests`. So any changes should probably be applied there as well.

Reply:
Hrmm.. I guess it is okay then. I was more thinking looking at the logs would be better and not spam the console, but this should be okay.